app/components/Views/AccountConnect/AccountConnectSingle/AccountConnectSingle.tsx

```diff
@@ -145,7 +131,7 @@ const AccountConnectSingle = ({
           type: accountAvatarType,
           accountAddress: address,
         }}
-        tagLabel={tagLabel}
+        tagLabel={tagLabel ? strings(tagLabel) : ''}
```

Comment:
Suggestion: for better separation of component responsibility, keep the fallback value handled in the `Cell` component and not in the `tagLabel` prop setting:
```suggestion
        {...(tagLabel && { tagLabel })}
```
Then in the Cell, display empty text, or do whatever, if the `tagLabel` prop is not define.

Reply:
hi, the only thing about here is `Cell` component is wrapper of `CellDisplay`, `CellMultiSelect` and `CellSelect` component,  and all three components will use `CellBase` component, however, i saw all the custom components defined `component-library` didnt contain any locale translation code ( `strings(tagLabel)`), if i do above change, then i need to handle translation in `CellBase` component which may break the pattern.